src/test/java/uk/nhs/digital/nhsconnect/lab/results/model/edifact/UnstructuredAddressTest.java

```diff
@@ -37,7 +38,17 @@ void testFromStringValidWithMinimumAddressFields() {
         final var address = UnstructuredAddress.fromString("ADR++US:LINE1::::++");
         assertAll(
             () -> assertThat(address.getFormat()).isEqualTo("US"),
-            () -> assertThat(address.getAddressLines()).containsExactly("LINE1", "", "", "", ""),
+            () -> assertThat(address.getAddressLines()).contains(new String[]{"LINE1", "", "", "", ""}),
+            () -> assertThat(address.getPostCode()).isEmpty()
+        );
+    }
+
+    @Test
+    void testFromStringValidWithNoPostcode() {
+        final var address = UnstructuredAddress.fromString("ADR++US:SLOANE SQUARE:LONDON");
```

Comment:
Maybe rename this test `testFromStringValidWithAddressFieldsAndNoPostcode`. Makes it look different from the test above

Reply:
As for similar suggestion above, fixed in 26459dba57b4aff2429f24cae805873125769cf9
Thanks!